account_invoice_refund_option/tests/test_account_invoice_refund_option.py

```diff
@@ -0,0 +1,31 @@
+from .common import AccountInvoiceRefundOptionTest
+from datetime import datetime
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class TestAccountInvoiceRefundOption(AccountInvoiceRefundOptionTest):
+
+    def test_account_invoice_refund_option(self):
+        """
+            Test the account invoice refund option to refund only selected
+            invoice line.
+        """
+        # Check invoice and invoice line
+        assert self.Invoice and self.Invoice.invoice_line_ids
+
+        # Create refund of selected invoice line
+        invoice_refund_wizard = self.account_invoice_refund_obj.create({
+            'date_invoice': datetime.today().strftime(
+                DEFAULT_SERVER_DATETIME_FORMAT),
```

Comment:
In odoo 12 date fields are python date objects. Why converting to string?
There's no need for `DEFAULT_SERVER_DATETIME_FORMAT`, is it?

Reply:
@ivantodorovich  : yes use default date object